README.md

```diff
@@ -1,130 +1,144 @@
+- A labeling application for tagging unidentified faces using [Streamlit](https://www.streamlit.io/).
+- Model training using [PyTorch](https://pytorch.org).
+- Automated model deployment using [Nuclio](https://nuclio.io/)
 
 <a id="demo-face-recognition-pipeline-output"></a>
-#### Pipeline Output
+### Pipeline Output
 
 <br><p align="center"><img src="./faces/workflow.png" width="500"/></p><br>
 
-<a id="demo-sklearn-pipe"></a>
-## Predictive Network/Telemetry Monitoring
-<!-- TODO: When the demo is read, edit the description, and remove the TBD. -->
-
-**TBD under construction**
+<a id="demo-churn"></a>
+## Churn Demo &mdash; Real-Time Customer-Churn Prediction (Telco Customer Churn Data Set)
 
-Demonstrate ingestion of telemetry data from simulator or live stream, feature exploration, data preparation (aggregation), model training, and automated model deployment.
+The [**chrun**](./churn/README.md) demo demonstrates analyses of customer-churn data using the Kaggle [Telco Customer Churn data set](https://www.kaggle.com/blastchar/telco-customer-churn), model training and validation using [XGBoost](https://xgboost.readthedocs.io), and model serving using real-time Nuclio serverless functions.
 
-The demo is maintained in a separate Git repository and also demonstrates how to manage project life cycle using git.
+The demo consists of few MLRun and Nuclio functions and a Kubeflow Pipelines orchestration:
 
-<br><p align="center"><img src="./docs/netops-metrics.png" width="500"/></p><br>
+1.  Write custom data encoders for processing raw data and categorizing or "binarizing" various features.
+2.  Summarize the data, examining parameters such as class balance and variable distributions.
+3.  Define parameters and hyperparameters for a generic XGBoost training function.
+4.  Train and test several models using XGBoost.
+5.  Identify the best model for your needs, and deploy it into production as a real-time Nuclio serverless function.
+6.  Test the model server.
 
-#### Pipeline Output
+### Pipeline Output
 
-<br><p align="center"><img src="./docs/netops-pipe.png" width="500"/></p><br>
+<br><p align="center"><img src="./churn/assets/pipeline-3.png" width="500"/></p><br>
 
-<a id="demo-churn"></a>
-## [Real-time Customer Churn Prediction (Kaggle Telco Churn dataset)](./churn/README.md)
+<a id="demo-netops"></a>
+## NetOps Demo &mdash; Predictive Network Operations/Telemetry
+<!-- TODO: When the demo is read, edit the description, replace "NetOps" at the
+  start of the description with the name of the demo directory and a link, 
+  remove the TBD note, and remove the [WORK IN PROGRESS] tag in the TOC. -->
 
-running customer churn data analyses using the **[Kaggle Telco Churn dataset](https://www.kaggle.com/blastchar/telco-customer-churn)**, training and validating an XGBoost model, and serving that with real-time Nuclio functions.
+> **Note:** This demos is currently work in progress.
```

Comment:
@Sharon-iguazio can remove the work in progress & TBD, should link to the netops project repo: https://github.com/mlrun/demo-network-operations

Reply:
As requested, I removed the WIP marks, linked the demo reference at the start of the description to the mlrun/demo-network-operations [README.md](https://github.com/mlrun/demo-network-operations/blob/master/README.md), and edited the internal TODO comment to ensure that if and when the demo is moved to the mlrun/demos repo, we update the README link (commit aa83d25).

Q: If the NetOps demo is ready, why is still in a separate repo and not in the mlrun/demos repo, like the other demos?